pkg/platform/kube/test/platform_test.go

```diff
@@ -908,7 +908,7 @@ func (suite *UpdateFunctionTestSuite) TestSanity() {
 				cmpopts.IgnoreFields(createFunctionOptions.FunctionConfig.Meta,
 					"ResourceVersion"), // kubernetes opaque value
 				cmpopts.IgnoreFields(createFunctionOptions.FunctionConfig.Spec,
-					"Image", "ImageHash"), // auto generated during deploy
+					"Image", "ImageHash", "Resources"), // auto generated during deploy
```

Comment:
the fact you are ignoring resources here means there's something broken in the flow. imo `UpdateFunction` is missing the resources enrichment.

Reply:
On the contrary - the `Resources` field is enriched during deployment, and since `createFunctionsOptions` does not have explicit resources the `Diff` is not empty.